Documentation/GeneralConventions/GuidelinesForImages.rst

```diff
@@ -120,7 +120,14 @@ Follow these guidelines
   corner radius of 2 px.
 * Text fields are TYPO3 Orange (#F49700) and have a corner radius of 2 px.
 * Dividing surfaces have a 2px frame in TYPO3 Orange (#F49700) with a corner radius
-  of 2 px
+  of 2 px.
+* Icon shadows have an
```

Comment:
Could you add a note that this is something from the design team and not just randomly decided?

Reply:
It is mentioned in the issue of this PR. Besides, all guidelines in the TYPO3 docs should be considered to be officially approved.